openspec/changes/archive/2026-03-30-autodetect-skill-suggestions/state.yaml

```
@@ -0,0 +1,13 @@
+change: 2026-03-30-autodetect-skill-suggestions
+current_phase: verify
+completed:
+  - explore
+  - propose
+  - spec
+  - design
+  - tasks
+  - apply
+  - verify
+next: verify
+next: archive
```

YAMLlint (1.38.0)

[error] duplication of key "next" in mapping

(key-duplicates)